### PR TITLE
refactored navigation tests

### PR DESCRIPTION
**Pull request by ngaruko**
# Description

[description]

medic/cht-core#[number]

# Code review items

- Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- Tested: Unit and/or e2e where appropriate
- Internationalised: All user facing text
- Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: tests/page-objects/common/common.po.js [hunk 1]
```diff
@@ -112,18 +119,33 @@ module.exports = {
...
   },
 
+  goToMessagesNative: async () => {
+    await browser.get(utils.getBaseUrl() + 'messages/');
+    await helper.waitUntilReady(medicLogo);
+    await helper.waitUntilReady(element(by.id('message-list')));
+  },
+
   goToPeople: async () => {
+    utils.deprecated('common.po.goToPeople', 'commmon.po.goToPeopleNative');
```
Comment (newtewt) [inline tests/page-objects/common/common.po.js:135]: These two methods look the same aside from the deprecated method. Do we need both?

File: tests/page-objects/common/common.po.js [hunk 2]
```diff
@@ -146,17 +168,53 @@ module.exports = {
...
+
   isAt: list => {
+    utils.deprecated('common.po.isAt', 'commmon.po.isAtNative');
     helper.waitUntilReady(medicLogo);
     return element(by.id(list)).isPresent();
   },
 
+  isAtNative: async (locator) => {
+    await helper.waitUntilReady(medicLogo);
+    return await element(by.css(locator)).isPresent();
```
Comment (newtewt) [inline tests/page-objects/common/common.po.js:215]: I would chose to either remove this `isAtNative` method and await it elsewhere or remove the return here.

**Review by newtewt — CHANGES_REQUESTED** [review]
Some small changes but otherwise looking good. I think you need to rebase as it seems like the diff is picking up changes that already exist. Might be github not refreshing when changes are pushed.

**Comment by dianabarsan** [comment]
Closing in favor of https://github.com/medic/cht-core/pull/6934

**Pull request closed without merging**